pkgs/development/python-modules/name-that-hash/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, poetry-core
+, click
+, rich
+}:
+
+buildPythonPackage rec {
+  pname = "nth";
```

The Python module should be named `name-that-hash`.

```suggestion
  pname = "name-that-hash";
```

Add it to `all-packages.nix` as `nth` with `toPythonApplication`.